challenges/objects-arrays.js

```diff
@@ -89,37 +131,60 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+zooAnimals.forEach(element => {
+  animalNames.push(
+    `Name: ${element.animal_name}, Scientific: ${element.scientific_name}`
+  );
+});
 console.log(animalNames);
 
+
 /* Request 2: .map()    
 
+MDN
+const map1 = array1.map(x => x * 2);
+
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
+const lowerCase = zooAnimals.map((zooAnimal, index, zooAnimals) => {
+  return zooAnimal.animal_name.toLowerCase();
+});
 console.log(lowerCase); 
 
 /* Request 3: .filter() 
 
+MDN
+const result = words.filter(word => word.length > 6);
+
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+const largerPopulation = zooAnimals.filter(zooAnimal => {
+  if (zooAnimal.population <= 5) {
+    return true
+  };
+});
 console.log(largerPopulation);
 
 /* Request 4: .reduce() 
 
+MDN
+const reducer = (accumulator, currentValue) => accumulator + currentValue;
+
 The zoos need to know their total animal population across the United States.  Find the total population from all the zoos using the .reduce() method.
 
 */
-const populationTotal = 0;
+const populationTotal = zooAnimals.reduce((population, zooAnimal, index, zooAnimals) => {
+  return population += zooAnimal.population;
+}, 0);
```

Comment:
No need for index or original array here either. And you can clean it up with an `implicit return`:
```
const populationTotal = zooAnimals.reduce((acc, el) =>  acc + el.population, 0);
```